src/System.Net.Security/src/Resources/Strings.resx

```diff
@@ -184,78 +240,9 @@
   <data name="net_invalid_enum" xml:space="preserve">
     <value>The specified value is not valid in the '{0}' enumeration.</value>
   </data>
-  <data name="event_EnumerateSecurityPackages" xml:space="preserve">
-    <value>Enumerating security packages:</value>
-  </data>
```

Comment:
Aren't all of these event_ resources used implicitly by the NetEventSource?

Reply:
What do you mean by "_implicitly_"? I looked for some names (`event_EnumerateSecurityPackages`, `event_CertsAfterFiltering`) in all sources and did not find them anywhere except `System.Net.Security`.

Reply:
I mean `EventSource` constructs these resource names dynamically by appending "event_" to the name of the event.